lib/chai/utils/type-detect.js

```diff
@@ -0,0 +1,18 @@
+const typesToLowerCase = [
+  "Number",
+  "String",
+  "Boolean",
+  "Null",
+  "Undefined",
+  "Function",
+  "Symbol",
+];
+
+export function type(obj) {
+  const type = Object.prototype.toString.call(obj).slice(8, -1);
```

Comment:
This could probably be simplified by checking `[Symbol.toStringTag]` on the object.